tcpdf.php

```diff
@@ -4283,7 +4283,7 @@ public function AddFont($family, $style='', $fontfile='', $subset='default') {
 			// all fonts must be embedded
 			$family = 'pdfa'.$family;
 		}
-		$tempstyle = strtoupper($style);
+		$tempstyle = strtoupper($style ?? '');
```

Comment:
(https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op) 
This syntax is not old php versions compatible 
```suggestion
		$tempstyle = strtoupper($style === null ? '' : $style);
```